pkg/server/cloud/aws/instances.go

```diff
@@ -255,11 +275,19 @@ func (e *AwsEC2) GetImage(spec cloud.BootImageSpec) (cloud.Image, error) {
 				creationTime = &ts
 			}
 		}
+		rootDeviceName, err := getRootDeviceName(img)
+		if err != nil {
+			klog.Warningf("getting root device name from image: %v", err.Error())
+		}
+		var rootDiskSize *int32
+		rootDiskSize = getRootDeviceVolumeSize(img.BlockDeviceMappings, rootDeviceName)
```

Comment:
Nitpicking: you don't need to declare the variable separately if you're assigning a value to it right away.

Reply:
nice catch:)